sphinx/domains/c.py

```diff
@@ -686,9 +686,15 @@ def describe_signature(self, signode: Any, mode: str,
 
 
 class ASTParameters(ASTBase):
-    def __init__(self, args: List[ASTFunctionParameter], attrs: ASTAttributeList) -> None:
+    def __init__(
+        self,
+        args: List[ASTFunctionParameter],
+        attrs: ASTAttributeList,
+        multiline: bool = False
```

Comment:
"multiline" -> "multi_line" please (only commented on one of them here)
```suggestion
        multi_line: bool = False
```